src/foam/u2/filter/FilterView.js

```diff
@@ -473,7 +472,58 @@ foam.CLASS({
             continue;
         }
       }
+      this.mementoUpdated = true;
```

Comment:
The slot won't be called if the value is always set to true, here we can do `this.memetoUpdated = ! this.mementoUpdated` that way the slot will always be called. We would also need to remove the if from the slot